### PR TITLE
Add Choral#findAllForUser()

### PR DESCRIPTION
**Pull request by cpjk**
Finds all chorals belonging to the given user

### CODE REVIEW

File: models/choral.js [hunk 1]
```diff
@@ -65,6 +65,20 @@ choralSchema.statics.createNew = function (attrs, cb) {
   });
 };
 
+choralSchema.statics.findAllForUser = function (user, cb) {
+  console.log("user id: " + user._id);
```
Comment (andreirtaylor) [inline models/choral.js]: Should we log this?

File: models/choral.js [hunk 2]
```diff
@@ -65,6 +65,20 @@ choralSchema.statics.createNew = function (attrs, cb) {
...
+  this.find({ userId: user._id }, (err, chorals) => {
+    if (err) {
+      cb(err, null);
+    }
+    else {
+      console.log("chorals");
```
Comment (andreirtaylor) [inline models/choral.js]: Probably shouldnt log this.

File: models/choral.js [hunk 3]
```diff
@@ -65,6 +65,20 @@ choralSchema.statics.createNew = function (attrs, cb) {
   });
 };
 
+choralSchema.statics.findAllForUser = function (user, cb) {
+  console.log("user id: " + user._id);
+  this.find({ userId: user._id }, (err, chorals) => {
+    if (err) {
```
Comment (andreirtaylor) [inline models/choral.js]: I really like the 

if (err) return cb(err, null);
cb(null, err);

rather than the if else. Just my opinion

Comment (cpjk) [reply models/choral.js]: lol yeah just did that before you commented :P

**Pull request merged**